### PR TITLE
Add a Custom 404 Page

### PR DESCRIPTION
**Pull request by steveyiyo**
Request add a Customer 404 Page, It can work fine if using GitHub Pages.
Please reference https://gohugo.io/templates/404/

Thanks

### CODE REVIEW

**Comment by steveyiyo** [comment]
The preview should looks like below

image

File: layouts/404.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{{ define "main"}}
+<main id="main">
+    <div>
+        <h1 id="title"><a href="{{ "/" | relURL }}">Go Home</a></h1>
```
Comment (Mitrichius) [inline layouts/404.html:4]: Need to support multilang for link, {{ .Site.BaseURL | relLangURL }}

File: layouts/404.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+        Sorry, This Page is not available.
```
Comment (Mitrichius) [inline layouts/404.html:5]: I think `This Page` should be without capitals

**Review by Mitrichius — CHANGES_REQUESTED** [review]
Thanks for PR.  
Some small changes are needed, and I will merge it.  

And after that, I will try to add i18n for messages :)

File: layouts/404.html (hunk 2 above)
Comment (steveyiyo) [reply layouts/404.html]: Sorry! I've fixed it.

File: layouts/404.html (hunk 1 above)
Comment (steveyiyo) [reply layouts/404.html]: Thanks for your feedback. I have updated it.

**Comment by steveyiyo** [comment]
Thanks for the feedback, I've updated it.

**Pull request merged**